helm/pattern-1/apim-with-analytics/templates/wso2apim-volume-claim.yaml

```diff
@@ -27,3 +27,21 @@ spec:
   selector:
     matchLabels:
       purpose: apim-shared-deployment
+
+---
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: wso2apim-with-analytics-tenants-volume-claim
```

Comment:
@mushthaq33 Apart from the Persistent Volume Claim you have introduced here, you need to introduce a Persistent Volume matching the labels defined in the selector and the resource request (storage) of the Claim, at [1].

And you may also have to introduce options to allow the user to edit NFS configurations via the values.yaml. Please see how this done for the shared mount between _<APIM_HOME>/repository/deployment/server_ directory from [2].

[1]: [Persistent Volume definition](https://github.com/wso2/kubernetes-apim/blob/ba885e0e133ca3995789a7d3745f06cdfa894753/helm/pattern-1/apim-with-analytics/templates/persistent-volumes.yaml)
[2]: [values.yaml](https://github.com/wso2/kubernetes-apim/blob/ba885e0e133ca3995789a7d3745f06cdfa894753/helm/pattern-1/apim-with-analytics/values.yaml#L7-L8)

Reply:
Fixed with e343483536bc9a81e58e186eb1e278dec401ee7a